### PR TITLE
add extra subgraphs

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

File: src/badger-nfts/abis/Gravity.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
+[
+    {
+        "constant": false,
```
Comment (axejintao) [inline src/badger-nfts/abis/Gravity.json:3]: nor relevant

File: src/badger-nfts/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Example Subgraph
+
+An example to help you get started with The Graph. For more information see the docs on https://thegraph.com/docs/.
```
Comment (axejintao) [inline src/badger-nfts/README.md:3]: remove

File: src/badger-nfts/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 The Graph
```
Comment (axejintao) [inline src/badger-nfts/LICENSE:3]: remove

File: src/badger-nfts/.gitignore [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
```
Comment (axejintao) [inline src/badger-nfts/.gitignore:34]: remove

File: src/badger-nfts/bin/Counter.bin [hunk 5]
```diff
@@ -0,0 +1 @@
+60806040526000808190555060f5806100196000396000f3fe6080604052600436106043576000357c0100000000000000000000000000000000000000000000000000000000900480633fa4f245146048578063d09de08a146070575b600080fd5b348015605357600080fd5b50605a6078565b6040518082815260200191505060405180910390f35b6076607e565b005b60005481565b600160008082825401925050819055507f20d8a6f5a693f9d1d627a598e8820f7a55ee74c183aa8f1a30e8d4e8dd9a8d846000546040518082815260200191505060405180910390a156fea165627a7a72305820fca3d11af59a6ac98027ec3bebdb10711f195860d6d9abd07921c88c8c50228f0029
```
Comment (axejintao) [inline src/badger-nfts/bin/Counter.bin:1]: remove?

**Review by mitche50 — APPROVED** [review]

**Pull request merged**